src/main/java/org/bptlab/cepta/Actor.java

```diff
@@ -0,0 +1,16 @@
+package org.bptlab.cepta;
+
+public class Actor{
+  public int id;
+  public String name;
+  public Actor(){
+
+  }
```

Comment:
Is this needed?

Reply:
Yes. It has something to do with how java treats some classes and others. For the code to work Actor needs a default constructor.